### ISSUE TITLE
Ticketing for hybrid events

### ISSUE THREAD

**Issue by dannybirchall** [wellcomecollection/wellcomecollection.org#7983]
TL;DR: We need to add ticketing links to individual events so that visitors can choose whether to attend a hybrid event online or in person

The full(er) story:

There's an innovation in the Pubic Programmes events plan for Jul-Sep, which is that some live events will be properly hybrid, with an in-person live audience, and an online audience, both of which will be capacity-limited and ticketed.  

Ticketing in eventbrite has to be separate for in-person and online, so there will be two sets of tickets to link to.

Can we change event pages so that there can be two eventbrite ticket links, one clearly labelled as in-person attendance and one as online attendance?

Some egs of how this works...

A single event page has a single eventbrite ticketing link
https://wellcomecollection.org/events/YnpPVxAAACIAI9Fp

which is set in the reservation tab, here:
https://wellcomecollection.prismic.io/documents~b=working&c=published&l=en-gb/YnpPVxAAACIAI9Fp/?section=Reservation

When you've got multiple iterations of the same event, there can be more than one eventbrite link on the same event page, eg
https://wellcomecollection.org/events/YmE20hAAAJ-4svJF

but they are set individually through sub-events through the schedule tab

https://wellcomecollection.prismic.io/documents~w=slalom&b=working&c=published&l=en-gb/YmE20hAAAJ-4svJF/?section=Schedule

Is it possible to have multiple instances of the scheduling fieldset on a single event, perhaps with the addition of a new field to each set for 'in person'/'online'?

**Comment by DominiqueMarshall** [comment]
@dannybirchall To reduce the amount of repeated text that happens currently with hybrid events, I wonder if it's possible to change the button copy so they're more explicit instead? Example below:



![Image](https://user-images.githubusercontent.com/60890140/169770813-cbd29d95-2d70-489b-8972-8803856aada6.jpg)

**Comment by dannybirchall** [comment]
@DominiqueMarshall that makes lots of sense to me. Might be an issue for @gestchild, as I think atm there's a default label for the button (ie you can't set it independently in the fieldset)

**Comment by DominiqueMarshall** [comment]
Ah that's a good point! If we can't change the button labels, then I think it could still be rendered a similar way by making sure the headings are really clear:



![Image](https://user-images.githubusercontent.com/60890140/169786401-f319df16-193d-478a-a856-865784258359.jpg)

**Comment by dannybirchall** [comment]
@DominiqueMarshall that looks good; though if it's easy to change the button labels I think that's preferable.

This is how we've done it the hacky way

https://wellcomecollection.org/events/YnOrDxAAACIAmAbF

-- I think there's definitely a balance to be struck here between making it really clear, and unnecessary repetition
